resourceDefinition.json

```diff
@@ -340,6 +340,17 @@
     "slug": "cog",
     "dashes": true
   },
+  {
+    "name": "communication_service",
+    "length": {
+      "min": 1,
+      "max": 63
+    },
+    "regex": "^(?=.{1,260}$)[a-zA-Z0-9][a-zA-Z0-9-]+$",
+    "scope": "resourceGroup",
+    "slug": "cog",
```

Comment:
This slug is being in use already.

Reply:
I am sorry, I copy paste and miss that. I push fix.